### DISCUSSION TITLE
Decimals in the plate

### DISCUSSION THREAD

**Discussion by kirin6666** [Backupiseasy/ThreatPlates#352] [Q&A] [answered]
Hey! I was wondering if u could guide me/tell me how to enable decimals in the % area of the plate? 
So if the target is on like 95.4% it will show it like that and not round it to 95, or if it's 95.7% rounding to 96%.
Is it possible and how?
Thanks in advance. 
Tried looking for a discord for contact but this is the closest I could get.

**Comment by Backupiseasy** [comment, marked as answer]
For that, you need LibDogTag and LibDogTag-Unit installed. You can then switch the status text to Custom and use something like

`[PercentHP:Percent]`

as status text.

There is no Discord for Threat Plates, so this way is fine :D

**Comment by kirin6666** [comment]
Thanks for the quick answer! Is that the only way to go for this? No option inside the addon itself?

**Comment by Backupiseasy** [comment]
Currently there is no other option to do that.
